w3/package.json

```diff
@@ -27,6 +27,8 @@
   "devDependencies": {
     "babel-core": "^6.26.0",
     "babel-loader": "^7.1.2",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
babel-preset-es2015 is redundant, babel-preset-env is sufficient

Reply:
I'll remove